identity/auth-starter/src/main/java/io/camunda/identity/authentication/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.authentication.config;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+import io.camunda.identity.authentication.handler.AuthFailureHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@Profile("auth-basic")
+@EnableWebSecurity
+@EnableMethodSecurity
+public class WebSecurityConfig {
+  public static final String[] UNAUTHENTICATED_PATHS =
+      new String[] {"/login**", "/logout**", "/error**", "/actuator**"};
+  private static final Logger LOG = LoggerFactory.getLogger(WebSecurityConfig.class);
+
+  @Bean
+  public SecurityFilterChain securityFilterChain(final HttpSecurity httpSecurity) {
+    try {
+      return httpSecurity.build();
+    } catch (final Exception e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  @Bean
+  @Primary
+  @Profile("auth-basic")
```

Comment:
Do we need to have this here as well as on the class level above? 🤔

Reply:
Good spot, that was left over from when we had multiple applicable beans in here, I'll remove